index.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="stylesheets/main.css" />
+
+    <title>2005 Toyota Corolla S Start Up/ Brief Walkthrough - YouTube</title>
+  </head>
+
+  <body>
+    <header id="main-header">
+      <div id="header-left-section">
+        <i class="fas fa-bars"></i>
+        <img id="logo" src="./images/youtube-logo.png" alt="Youtube Logo" />
+      </div>
+      <div id="header-middle-section">
+        <form action="#" id="search-form">
+          <input type="text" placeholder="Search" />
+          <button><i class="fas fa-search"></i></button>
+        </form>
+      </div>
+
+      <div id="header-right-section">
+        <i class="fas fa-video"></i>
+        <i class="fas fa-th"></i>
+        <i class="fas fa-bell"></i>
+        <img
+          src="https://yt3.ggpht.com/a/AATXAJxi-vx7V6HLPGGsFWw_PGiuSs_IRi1B-YAj2A=s176-c-k-c0xffffffff-no-rj-mo"
+          alt=""
+        />
+      </div>
+    </header>
+    <main class="container" id="videos">
+      <section id="left-section">
+        <section id="main-video">
+          <section id="video">
+            <iframe
+              width="560"
+              height="315"
+              src="https://www.youtube.com/embed/BDtGXMhiooM"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </section>
+
+          <footer>
```

Comment:
Kindly remove the ```<footer>```, that is not the right position. Footers are to be at the bottom of the page

Reply:
Hello @addod19, thank you for the quick feedback. We have worked on the changes suggested.